src/main/scala/util/RunUtils.scala

```diff
@@ -587,11 +587,14 @@ object RunUtils {
       ).resolveIndirectCalls()
       */
 
-      modified = transforms.VSAIndirectCallResolution(
-        ctx.program,
-        result.vsaResult,
-        result.mmmResults
-      ).resolveIndirectCalls()
+      if analysisResult.size < 2 then
+        modified = true
+      else
+        modified = transforms.VSAIndirectCallResolution(
+          ctx.program,
+          result.vsaResult,
+          result.mmmResults
+        ).resolveIndirectCalls()
```

Comment:
What's the reason for this, why do we not try to resolve indirect calls until at least two iterations have been completed?